.github/workflows/integrationTest.yml

```diff
@@ -0,0 +1,195 @@
+name: Deploy app to ubuntu server
+
+on: push
```

Comment:
The repo should block all not authorized users from pushing to a branch. This will allow testing on forked repo using only the forked repo's secrets.

Reply:
I have not looked at the security blocks for branch of this repo that is just my assumption based on experience with repos.